project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=1.0.3
+sbt.version=1.1.0-RC1
```

Comment:
1.1 adds LSP support, so you can get error highlighting and jump-to-class in VS-Code. Dale and Eugene say it's good to go so I don't think there's any harm in upgrading.

Reply:
Should we wait for the final release. It should be quite soon
I want to add scalameta support to ocs3 along 1.1.0. WDYT?

Reply:
Sure, I can roll this back to 1.0.4 for now.